src/server/routes/configuration.js

```diff
@@ -3,14 +3,12 @@
 const express = require('express')
 const router = express.Router()
 const { requireUser } = require('../access-helpers')
-const { user: getUser, users: getUsers, roles: getRoles } = require('../db/users')
+const { roles: getRoles } = require('../db/users')
 
 router.get('/', requireUser, async function (req, res) {
-  const user = await getUser(req.signedCookies.userId)
-  const users = user.role === 'admin' ? await getUsers() : [user]
   const roles = await getRoles()
 
-  res.json({ configuration: { users, roles } })
+  res.json({ configuration: { roles } })
```

Comment:
we can almost kill this `configuration` route -- we should just keep track of roles in the store directly, vs as the only remaining child of the configuration object.